README.md

```diff
@@ -2,10 +2,14 @@
 
 ## Installation
 
-Run:
+Using pip:
+
+    pip install git+https://github.com/moddevices/mod-devel-cli
```

Comment:
We should take the chance and update the URL to be mod-audio github org, instead of moddevices.
Redirect still works, but undefined until how long